src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -205,26 +243,26 @@ class ZWaveJSConfigDashboard extends LitElement {
                     ${this._network.client.ws_server_url}<br />
                   </div>
                   <div class="card-actions">
-                    <mwc-button
-                      @click=${this._removeNodeClicked}
-                      .disabled=${this._status === "connecting"}
-                    >
-                      ${this.hass.localize(
-                        "ui.panel.config.zwave_js.common.remove_node"
-                      )}
-                    </mwc-button>
+                    ${this._status === "connected" &&
+                    this._network?.controller.inclusion_state ===
+                      InclusionState.Idle
+                      ? html`
```

Comment:
Why hide this now instead of disable?

Reply:
good idea

Reply:
I wasn't able to figure out how to disable. I tried the .disabled property but it didn't solve the problem. Maybe I missed something obvious?

Reply:
This should do it:
```ts
.disabled=${this._status !== "connected" || this._network.controller.inclusion_state !== InclusionState.Idle}
```